common/app-starters-task-launch-request-common/src/main/java/org/springframework/cloud/stream/app/tasklaunchrequest/DataFlowTaskLaunchRequestAutoConfiguration.java

```diff
@@ -50,10 +51,16 @@ public DataFlowTaskLaunchRequestAutoConfiguration(DataflowTaskLaunchRequestPrope
 	}
 
 	@Bean
-	public TaskLaunchRequestTransformer taskLaunchRequestTransformer() {
+	public Function<Message<?>, Message<DataFlowTaskLaunchRequest>> taskLaunchRequest() {
 		return message -> dataflowTaskLaunchRequest(message);
 	}
 
+	@Deprecated
```

Comment:
Why can't we just remove this altogether?

Reply:
Currently SFTP dataflow source needs that. I plan to refactor that later, then we can remove this.